garrysmod/gamemodes/terrortown/gamemode/cl_equip.lua

```diff
@@ -436,7 +434,7 @@ concommand.Add("ttt_cl_traitorpopup_close", ForceCloseTraitorMenu)
 
 function GM:OnContextMenuOpen()
    local r = GetRoundState()
-   if r == ROUND_ACTIVE and not (LocalPlayer():GetTraitor() or LocalPlayer():GetDetective()) then
+   if r == ROUND_ACTIVE and not LocalPlayer():GetTraitor() or LocalPlayer():GetDetective() then

```

Comment:
not only does this change the meaning of the if condition, but removing parenthesis like this is usually a negative change. the parenthesis were there in the first place to increase clarity.